src/tracer.js

```diff
@@ -133,6 +133,11 @@ export const trace = ({
     eventFilter,
   });
 
+  if (context.wrapped_by_lumigo) {
+    return userHandler(event, context, callback);
+  }
+  context.wrapped_by_lumigo = true;
```

Comment:
the user may see this, maybe we should start at `__` to make clear it's our private